revel.go

```diff
@@ -94,3 +132,50 @@ func getRevelLevel(lev mappers.Level) loggers.Standard {
 		panic("unreachable")
 	}
 }
+
+// shortenFile returns the folder and file name of an absolute file path.
+func shortenFile(file string) string {
+	short := file
+	foundOne := false
+	for i := len(file) - 1; i > 0; i-- {
+		if file[i] == '/' {
+			if !foundOne {
+				foundOne = true
+				continue
+			}
+			short = file[i+1:]
+			break
+		}
+	}
+	return short
+}
+
+// stackTrace is a bit set for enabling stack traces.
+var stackTrace int
```

Comment:
It was purely an addition, so it would be possible to selectively enable forced stack traces on certain error levels.